internal/repository/request.go

```diff
@@ -0,0 +1,14 @@
+package repository
+
+type GetRequest struct {
```

Comment:
We need to keep this generic, not feasible to be done for multiple commands getting generated.